features/claim/claimForm.tsx

```diff
@@ -63,10 +63,18 @@ export const ClaimForm: FC = () => {
   const handleClaim = useCallback(
     async (data: ClaimFormData) => {
       const { amount, address } = data;
-      await claim(amount, address);
-      resetCache();
+      try {
+        await claim(amount, address);
+        resetCache();
+        setValue('amount', '');
```

Comment:
if we are clearing the field AFTER async actions happens better to lock the field until it is ended.
otherwise, claim should be after everything else, I guess

Reply:
It's a good catch, added loading state (added screenshot to description)